icubam/time_utils.py

```diff
@@ -44,7 +44,7 @@ def localewise_time_ago(ts, locale, ts_reference=None):
   return template.format(delta=count, units=units)
 
 
-def parse_hour(hour, sep=':') -> Tuple[str, str]:
+def parse_hour(hour, sep=':') -> Tuple[Union[int, str], ...]:
```

Comment:
when too complex, i d rather skip it

Reply:
Reverted. Complex types often mean that the API has an issue. Here, this function returns a `Tuple(int, int)` or `("", "")` I guess could be done better.